uploadNeuroDB/NeuroDB/objectBroker/MriScannerOB.pm

```diff
@@ -0,0 +1,120 @@
+package NeuroDB::objectBroker::MriScannerOB;
+
+=pod
+
+=head1 NAME
+
+NeuroDB::objectBroker::MriScannerOB -- An object broker for mri_scanner records
+
+=head1 SYNOPSIS
+
+  use NeuroDB::Database;
+  use NeuroDB::objectBroker::MriScannerOB;
+  use TryCatch;
```

Comment:
missing use NeuroDB::DatabaseException and NeuroDB::objectBroker::ObjectBrokerException.

Reply:
Right. Corrected.